core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -24,6 +24,29 @@ var ExplorationEditorTranslationTab = function() {
   var translationWelcomeModal = element(
     by.css('.protractor-test-translation-tab-welcome-modal'));
 
+  this.saveUploadedAudioButton = element(
+    by.className('protractor-test-save-button'));
+
+  this.errorMessage = element(by.css('div.error-message'));
+
+  this.colorOfFirstState = function() {
+    return (element.all(by.css(
```

Comment:
Can't you inline these element.(....) calls? Why put it in a function? (ditto below)

Reply:
Putting these functions inline throws an error. It shows that no such element found.
Keeping these as it is. :)

Reply:
Ya. Welcome. :)